### ISSUE TITLE
Quasar crash: IMBD Top 250

### ISSUE THREAD

**Issue by florinandrus** [scakemyer/plugin.video.quasar#313]
```
19:11:58 T:2882532256  NOTICE: [plugin.video.quasar] Requesting http://localhost:65251/movies/imdb250 from ['plugin://plugin.video.quasar/movies/imdb250', '13', '']
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] panic: Bad status: 429
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar]
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] goroutine 941 [running]:
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] panic(0xa04278, 0x112fc4b0)
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/usr/local/go/src/runtime/panic.go:464 +0x330
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/tmdb.GetMovieById.func1()
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/tmdb/movie.go:50 +0x428
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/util.(*RateLimiter).Call(0x11256a80, 0x1141bf54)
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/util/ratelimit.go:45 +0xa0
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/tmdb.GetMovieById(0x113e044c, 0x4, 0x11568488, 0x2, 0x361c14)
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/tmdb/movie.go:55 +0x2b8
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/tmdb.GetMovie(0x556, 0x11568488, 0x2, 0x0)
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/tmdb/movie.go:26 +0x4c
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] github.com/scakemyer/quasar/tmdb.GetMovies.func1(0x113e0000, 0x112da800, 0xfb, 0xfb, 0x11568488, 0x2, 0xc7, 0x556)
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/tmdb/movie.go:77 +0x5c
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] created by github.com/scakemyer/quasar/tmdb.GetMovies
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] 	/go/src/github.com/scakemyer/quasar/tmdb/movie.go:78 +0x130
19:12:01 T:2882532256   ERROR: [plugin.video.quasar] Traceback (most recent call last):
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/root/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 107, in run
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]     data = _json(url)
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/root/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 48, in _json
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]     with closing(urllib2.urlopen(url)) as response:
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 431, in open
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 449, in _open
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 1227, in http_open
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 1200, in do_open
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/httplib.py", line 1132, in getresponse
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/httplib.py", line 453, in begin
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/httplib.py", line 417, in _read_status
19:12:01 T:2882532256   ERROR: [plugin.video.quasar] BadStatusLine: ''
19:12:01 T:2882532256   ERROR: [plugin.video.quasar]
19:12:01 T:2882532256   DEBUG: POParser: loaded 116 strings from file /root/.kodi/addons/plugin.video.quasar/resources/language/English/strings.po
19:12:01 T:2882532256    INFO: CPythonInvoker(40, /root/.kodi/addons/plugin.video.quasar/navigation.py): script successfully run
19:12:01 T:3034574848   DEBUG: ------ Window Init (DialogKaiToast.xml) ------
19:12:01 T:3034574848   DEBUG: Window DialogKaiToast.xml was already loaded
19:12:01 T:3034574848   DEBUG: Alloc resources: 0.02ms
19:12:01 T:2882532256    INFO: Python script stopped
19:12:01 T:2882532256   DEBUG: Thread LanguageInvoker 2882532256 terminating
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] System information: linux_arm 3.10.96
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] Kodi build version: 15.2 Git:0a9cb7b2d1
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] OS type: Linux
19:12:01 T:2652894112  NOTICE: [plugin.video.quasar] uname: ('Linux', 'dsPlus', '3.10.96', '#1 SMP PREEMPT Fri Feb 19 01:33:32 CET 2016', 'armv7l', '')
19:12:01 T:3005215648   DEBUG:  WaitOnScriptResult - plugin exited prematurely - terminating
19:12:01 T:3034574848   ERROR: GetDirectory - Error getting plugin://plugin.video.quasar/movies/imdb250
19:12:01 T:3034574848   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.quasar/movies/imdb250) failed
19:12:01 T:3034574848   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.quasar/movies/)
19:12:01 T:3034574848   DEBUG:   ParentPath = [plugin://plugin.video.quasar/]
19:12:01 T:3034574848   DEBUG: Loading items: 33, directory: plugin://plugin.video.quasar/movies/ sort method: 0, ascending: false
19:12:01 T:2882532256  NOTICE: Thread BackgroundLoader start, auto delete: false
19:12:01 T:3034574848   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
19:12:01 T:3034574848   DEBUG: ------ Window Deinit (DialogKaiToast.xml) ------
19:12:01 T:2882532256   DEBUG: Thread BackgroundLoader 2882532256 terminating
19:12:02 T:3034574848   DEBUG: ------ Window Init (DialogKaiToast.xml) ------
19:12:02 T:3034574848   DEBUG: Window DialogKaiToast.xml was already loaded
19:12:02 T:3034574848   DEBUG: Alloc resources: 0.00ms
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] quasard: starting quasard
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] Deleting stale files set([])
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m________
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____  \  __ _______    ___________ _______
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m /  / \  \|  |  \__  \  /  ___/\__  \\_  __ \
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m/   \_/.  \  |  // __ \_\___ \  / __ \|  | \/
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m\_____\ \_/____/(____  /____  >(____  /__|
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m       \__>          \/     \/      \/
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0m
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  main         ▶ main             [0mVersion: "v0.9.23" Go: go1.6
19:12:04 T:2652894112  NOTICE: [plugin.video.quasar] INFO  config       ▶ Reload           [0mReloading configuration...
```
full log: http://pastebin.com/MGwx4fWe

**Comment by KillerJoeBR** [comment]
It's happening for me too

**Comment by ShlomiD83** [comment]
same here..

**Comment by m4xc4v413r4** [comment]
Now do everyone a favor and put that on kodilogs for example instead of pasting a wall of text here.

**Comment by scakemyer** [comment]
429 means rate limit exceeded. I've noticed that on IMDB top 250 also, made the rate limiter go easier and it worked, but it might still be too close to the limit. Try waiting a few minutes, but I'll eventually fix it properly.

**Comment by scakemyer** [comment]
Related to #279

**Comment by scakemyer** [comment]
Closed by scakemyer/quasar@1174eae
